microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
 
-   ![The report for the Audits panel of the performance of the site](../media/speed-glitch-tony-remix-audits-performance-metrics-collapsed.msft.png)
+![The Metrics section](./images/metrics-section.png)
 
-#### Handling report errors
+Click **Expand view** to display a description for each metric.  Then click **Learn More** to read documentation about it.
 
-If you ever get an error in your Audits panel report, try running the demo tab from an **InPrivate** window with no other tabs open.  This ensures that you are running Microsoft Edge from a clean state.  Microsoft Edge Extensions in particular often interfere with the auditing process.
+![The expanded metrics section](./images/metrics-learn-more.png)
 
-<!--todo: add screen capture for error in audit -->
-<!--
-![A report that errored out](../media/speed-.msft.png)
--->
+###### Screenshots
 
-### Understand your report
+Below the **Metrics** section is a collection of screenshots that show you how the page looked as it loaded.
 
-The number at the top of your report is the overall performance score for the site.  Later, as you make changes to the code, the number displayed should rise.  A higher score means better performance.
+![Series of screenshots showing how the page looked while loading](./images/report-screenshots.png)
 
-![The overall performance score](../media/speed-glitch-tony-remix-audits-performance-metrics-collapsed-metrics-highlighted.msft.png)
+###### Opportunities
 
-The **Metrics** section provides quantitative measurements of the performance of the site.  Each metric provides insight into a different aspect of the performance.  For example, **First Contentful Paint** tells you when content is first painted to the screen, which is an important milestone in the user's perception of the page load, whereas **Time To Interactive** marks the point at which the page appears ready enough to handle user interactions.
+The **Opportunities** section provides specific tips on how to improve the load performance of this specific webpage:
 
-![The Metrics section](../media/speed-glitch-tony-remix-audits-performance-metrics-collapsed-highlighted.msft.png)
+![The Opportunities section](./images/opportunities-section.png)
 
-Click the highlighted toggle button in the following figure to display a description for each metric.  Then click **Learn More** to read documentation about it.
+Click an opportunity to display more information about it, and then click **Learn more** to read documentation about why an opportunity is important, and specific recommendations on how to fix it:
```

Comment:
```suggestion
Click an opportunity to display more information about it, and then click **Learn more** to read about why an opportunity is important, and specific recommendations on how to fix it:
```
"documentation" is implicit and i try to keep the spotlight on tech, not on docs (try to write words about tech, not about docs)